src/csharp/Grpc.Core.Api/ServerCallContext.cs

```diff
@@ -135,7 +142,9 @@ public WriteOptions WriteOptions
         protected abstract Status StatusCore { get; set; }
         /// <summary>Provides implementation of a non-virtual public member.</summary>
         protected abstract WriteOptions WriteOptionsCore { get; set; }
-          /// <summary>Provides implementation of a non-virtual public member.</summary>
+        /// <summary>Provides implementation of a non-virtual public member.</summary>
         protected abstract AuthContext AuthContextCore { get; }
+        /// <summary>Provides implementation of a non-virtual public member.</summary>
+        protected abstract IDictionary<object, object> UserStateCore { get; }
```

Comment:
Adding an abstract property is a breaking change. For backwards compat this would need to be virtual with a default implementation.

Reply:
Doh. Good catch.